### PR TITLE
feat: add datadetect adopters

### PR DESCRIPTION
**Pull request by jfcurran**
- adopters logo added

### CODE REVIEW

**Comment by jfcurran** [comment]
link does not work with https

File: src/components/adopters.tsx [hunk 1]
```diff
@@ -70,6 +71,11 @@ const adopters = [
     image: spiribo,
     url: 'https://www.spiri.bo/',
   },
+  {
+    title: 'Data Detect',
+    image: datadetect,
+    url: 'http://datadetect.com/',
```
Comment (zepatrik) [inline src/components/adopters.tsx:77]: Then use e.g. 
```suggestion
    url: 'https://unifiedglobalarchiving.com/data-detect/',
```
but we really can't have non https links on our website.

**Review by zepatrik — CHANGES_REQUESTED** [review]

File: src/components/adopters.tsx (hunk 1 above)
Comment (vinckr) [reply src/components/adopters.tsx:77]: ok I will talk to orly if that is good too. 
he told me to use the datadetect.com cause they might switch to a dedicated website for it

**Comment by vinckr** [comment]
@jfcurran 
Also I cant see the logo in the preview? 
is that correct?

File: src/components/adopters.tsx (hunk 1 above)
Comment (zepatrik) [reply src/components/adopters.tsx:77]: ok but it won't help them now if we have the http URL there, they should add a redirect over https as well

**Comment by jfcurran** [comment]
> @jfcurran
> Also I cant see the logo in the preview?
> is that correct?

Yes, this is correct. it is visible in our project pages

**Comment by vinckr** [comment]
a https redirect will be added. will be soon probably..

**Comment by aeneasr** [comment]
Is this ready for merge?

**Comment by jfcurran** [comment]
> Is this ready for merge?

https:// has not been implemented yet - it is not ready yet.
@vinckr

**Comment by vinckr** [comment]
He wanted to let me know when they implement the https redirect. 
If its not ready by tomorrow I will ask again. 

If its a problem to leave the PR open we can use https://unifiedglobalarchiving.com/data-detect/ in the meantime and then change, but I would rather do the correct URL right away

**Comment by vinckr** [comment]
@aeneasr 
Ok I just talked to orly and we can merge with  ` url: 'https://unifiedglobalarchiving.com/data-detect/',`

He will let me know when they change the url, but it could take a little longer...

**Review by zepatrik — APPROVED** [review]

**Pull request merged**